fastify.js

```diff
@@ -794,6 +794,7 @@ function fastify (options) {
       // only call isAsyncConstraint once
       if (isAsync === undefined) isAsync = router.isAsyncConstraint()
       if (rewriteUrl) {
+        req.originalUrl = req.originalUrl || req.url
```

Comment:
This will give you always `req.url` as we do not know yet the `originalUrl` because is calculated on the next step.
Maybe we should just:

```suggestion
				const originalUrl = req.url
        req.originalUrl = originalUrl
```

wdyt?